tests/test_api_build.py

```diff
@@ -1945,7 +1946,7 @@ def test_add_pip_as_python_dependency_from_condarc_file(
     testing_metadata, testing_workdir, add_pip_as_python_dependency, monkeypatch
 ):
     """
-    Test whether settings from .condarc files are heeded.
+    Test whether settings from .condarc files are needed.
```

Comment:
```suggestion
    Test whether settings from .condarc files are heeded.
```

I think this wording edit needs to be reverted, since this test is checking whether or not the settings in the `.condarc` files are actually being taken into account/paid attention to.